modules/oneVideoBidAdapter.js

```diff
@@ -179,6 +186,23 @@ function getRequestData(bid) {
   if (bid.params.site && bid.params.site.id) {
     bidData.site.id = bid.params.site.id
   }
+
+  if (isConsentRequired(consentData)) {
+    bidData.regs = {
+      ext: {
+        gdpr: NUMERIC_VALUES.TRUE
```

Comment:
Can you clarify what `NUMERIC_VALUES` is meant to be?  I've seen other adapters pass a `1` or `0` for the boolean value of the `gdprApplies` in this field.  Is that what you want to do as well?

Currently this is going to return an `undefined` error, since I don't see it declared/imported from anywhere.  Can you update this part?